src/index.html

```diff
@@ -11,8 +11,62 @@
       rel="stylesheet"
       href="./styles/index.scss"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@100;400;500;700&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card_imac">
+        <img
+          class="card_imac_img"
+          src="../src/images/imac.jpeg"
+          alt="imac_photo"
+        />
+      </div>
+      <div class="card_info">
+        <h1 class="card_text_name">
```

Comment:
Avoid using <br> to create space between lines. Use CSS margin or padding instead.